HowTos/AviatrixLogging.rst

```diff
@@ -325,6 +326,17 @@ Example log:
   2019-11-19T09:40:12.878075-08:00 ip-172-31-93-101 cloudxd: AviatrixFireNet: Firewall i-021f23187b8ac81c9~~tran-fw-1 in FireNet VPC vpc-0f943cd05455358ac~~cal-transit-vpc-1 state has been changed to up.
 
 
+AviatrixVPNVersion
+-------------------
+
+Log messages with this prefix come from the Controller whenever it rejects an Aviatrix VPN client connection.
```

Comment:
come from the "gateway"

Reply:
The log is in the controller as the controller decides to reject it, then ask gw to do it

Reply:
This should have been logged in gateway